src/components/AdmissionsDash.js

```diff
@@ -105,11 +107,11 @@ export class AdmissionsDash extends React.Component {
     if (option.value === "default") {
```

Comment:
I don't know what this code does but can't we rewrite it as :
`if (option.value === 'default') {
    this.stage = null;
    this.value = "Student Details";
   } else {
    this.stage = option.value;
  }
  this.fetchStudents(filterValues, this.value)
`

Reply:
I don't know Saquib. Komal has written this code.